docker/examples/Dockerfile

```diff
@@ -1,14 +1,6 @@
-# Bulding on top of deploy image, this Dockerfile adds libraries needed
-# for running examples.
+# The examples Docker image adds dependencies needed to run the examples
 
-FROM ray-project/ray:deploy
-
-# Tensorflow
-RUN pip install --upgrade https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.9.0-cp27-none-linux_x86_64.whl
-
-# SciPy
-RUN pip install scipy
-
-# Gym
-RUN sudo apt-get -y install zlib1g-dev libjpeg-dev xvfb libav-tools xorg-dev python-opengl libsdl2-dev swig wget
+FROM ray-project/deploy
+RUN conda install -y -c conda-forge tensorflow
```

Comment:
This is the suggested command for installation on Anaconda given on the Tensorflow page:
https://www.tensorflow.org/get_started/os_setup#anaconda_installation

I had tried using the pip install but encountered errors.